pyoptsparse/pyParOpt/ParOpt.py

```diff
@@ -257,28 +262,28 @@ def evalObjConGradient(self, x, g, A):
                 else:
                     opt.setNormType(_ParOpt.L2_NORM)
 
-                # Initialize the problem
-                tr.initialize()
+                # Set the ParOpt options
+                self._set_paropt_options(opt)
```

Comment:
Is this call necessary, or a duplicate of line 255? And since the `setNormType` call is already in the `set_paropt_options` call, do they have to be set again here?

Reply:
@gjkennedy can you take a look at this when you've got a chance? And the same with the `tr_opt_abs_tol` value above.

Reply:
The norm_type is set in set_paropt_options so it can be removed here.

Reply:
Yeah I agree. Also, `_set_paropt_options` is called twice, on L255 and L266, which I think is redundant, right?